wiki/Tournaments/GBC/GBC_2022_Spring/en.md

```diff
@@ -135,30 +138,48 @@ The GB Cup 2022 Spring was run by the GB Team and various community members.
-| 1 | 金戈铁马的鲜丰水果摊 | 0-0 | 0 | 0 | 0 | 倒霉蛋 |  |  |
-| 2 | 你没对象对不队 | 0-0 | 0 | 0 | 0 | 15 |  |  |
-| 3 | 涩涩不队 | 0-0 | 0 | 0 | 0 | 吸手手 |  |  |
-| 4 | Yzhhのqlll森林汪酱不会大叠 | 0-0 | 0 | 0 | 0 | 理塘 |  |  |
-| 5 | 可爱的hyahya酱上线啦！！ | 0-0 | 0 | 0 | 0 | 啊对对 |  |  |
-| 6 | 货拉拉拉不拉拉布拉多队 | 0-0 | 0 | 0 | 0 | 啊不对 |  |  |
-| 7 | bikabika | 0-0 | 0 | 0 | 0 | 摩卡 |  |  |
-| 8 | Team MyAngel Koishi | 0-0 | 0 | 0 | 0 | 大怨种 |  |  |
-| 9 | 大怨种被迫当队长 | 0-0 | 0 | 0 | 0 | TMK |  |  |
-| 10 | 青叶摩卡的百合后花园 | 0-0 | 0 | 0 | 0 | bika |  |  |
-| 11 | 啊不对不队 | 0-0 | 0 | 0 | 0 | 货拉拉 |  |  |
-| 12 | 啊对对队 | 0-0 | 0 | 0 | 0 | hya酱 |  |  |
-| 13 | 太美丽了理塘 | 0-0 | 0 | 0 | 0 | Yzh |  |  |
-| 14 | 皮蛋蛋吸手手俱乐部 | 0-0 | 0 | 0 | 0 | 涩涩 |  |  |
-| 15 | Team 15 | 0-0 | 0 | 0 | 0 | 没对象 |  |  |
-| 16 | 头号倒霉蛋 | 0-0 | 0 | 0 | 0 | 水果摊 |  |  |
-
-¹ Seed\
-² Matches\
-³ Beatmaps\
-⁴ Beatmap difference\
-⁵ Buchholz score\
+| 1 | 金戈铁马的鲜丰水果摊 | 1-0 | 6-1 | +5 | -1 | **6-1**</br>倒霉蛋 | hya酱 |  |
+| 2 | 可爱的hyahya酱上线啦！！ | 1-0 | 6-1 | +5 | -1 | **6-1**</br>啊对对 | 水果摊 |  |
+| 3 | 青叶摩卡的百合后花园 | 1-0 | 6-1 | +5 | -1 | **6-1**</br>bika | 理塘 |  |
+| 4 | 太美丽了理塘 | 1-0 | 6-1 | +5 | -1 | **6-1**</br>Yzh | 摩卡 |  |
+| 5 | GBC最强阵容9个Miyes | 1-0 | 6-2 | +4 | -1 | **6-2**</br>没对象 | 大怨种 |  |
+| 6 | 大怨种被迫当队长 | 1-0 | 6-3 | +3 | -1 | **6-3**</br>TMK | 最强阵 |  |
+| 7 | 货拉拉拉不拉拉布拉多队 | 1-0 | 6-4 | +2 | -1 | **6-4**</br>啊不对 | 涩涩 |  |
+| 8 | 涩涩不队 | 1-0 | 6-5 | +1 | -1 | **6-5**</br>吸手手 | 货拉拉 |  |
+| 9 | 皮蛋蛋吸手手俱乐部 | 0-1 | 5-6 | -1 | +1 | *5-6*</br>涩涩 | 啊不对 |  |
+| 10 | 啊不对不队 | 0-1 | 4-6 | -2 | +1 | *4-6*</br>货拉拉 | 吸手手 |  |
+| 11 | Team MyAngel Koishi | 0-1 | 3-6 | -3 | +1 | *3-6*</br>大怨种 | 没对象 |  |
+| 12 | 你没对象对不队 | 0-1 | 2-6 | -4 | +1 | *2-6*</br>最强阵 | TMK |  |
+| 13 | Yzhhのqlll森林汪酱不会大叠 | 0-1 | 1-6 | -5 | +1 | *1-6*</br>理塘 | bika |  |
+| 14 | bikabika | 0-1 | 1-6 | -5 | +1 | *1-6*</br>摩卡 | Yzh |  |
+| 15 | 啊对对队 | 0-1 | 1-6 | -5 | +1 | *1-6*</br>hya酱 | 倒霉蛋 |  |
+| 16 | 头号倒霉蛋 | 0-1 | 1-6 | -5 | +1 | *1-6*</br>水果摊 | 啊对对 |  |
+
+¹ Seed, Top 8 Advanced\
+² Beatmap difference\
+³ Buchholz score
+
+#### Regular stage week 1
+
+Saturday, April 9, 2022
+
+| Team A | | | Team B | Match link |
+| --:  |--: | :-- | :-- | :-: |
+| bikabika | 1 | **6** | **青叶摩卡的百合后花园** | [#1](https://osu.ppy.sh/community/matches/99564304) |
+| **可爱的hyahya酱上线啦！！** | **6** | 1 | 啊对对队 | [#1](https://osu.ppy.sh/community/matches/99566208) |
+| **金戈铁马的鲜丰水果摊** | **6** | 1 | 头号倒霉蛋 | [#1](https://osu.ppy.sh/community/matches/99567449) |
+| **涩涩不队** | **6** | 5 | 皮蛋蛋吸手手俱乐部 | [#1](https://osu.ppy.sh/community/matches/99568857) |
+
+Sunday, April 10, 2022
+
+| Team A | | | Team B | Match link |
+| --:  |--: | :-- | :-- | :-: |
+| Team MyAngel Koishi | 3 | **6** | **大怨种被迫当队长** | [#1](https://osu.ppy.sh/community/matches/99603315) |
+| **货拉拉拉不拉拉布拉多队** | **6** | 4 | 啊不对不队 | [#1](https://osu.ppy.sh/community/matches/99606894) |
+| Yzhhのqlll森林汪酱不会大叠 | 1 | **6** | **太美丽了理塘** | [#1](https://osu.ppy.sh/community/matches/99608620), [Screenshot](GBC2_Spr_R1G7_YzH_vs_理塘.png) |
```

Comment:
was the match history set to private when creating the lobby? you can link to a third party image host to preserve image quality

Reply:
yes so i want to add a screenshot for ref , since image quality is not important I decided to lower the quality (make file size < 500 kb)